libraries/rush-lib/src/logic/base/BaseInstallManager.ts

```diff
@@ -741,6 +834,38 @@ export abstract class BaseInstallManager {
         );
       }
     }
+
+    if (this._rushConfiguration.hasSplitWorkspaceProject) {
```

Comment:
If `hasSplitWorkspaceProject=false`, maybe we should delete `common/temp-split` to avoid confusion